File: coding/17th May 2025/Count the Number of Consistent Strings.java
//Count the Number of Consistent Strings
class Solution 
{
    public int countConsistentStrings(String allowed, String[] words) 
    {
        int res = 0;
        for(String str : words)
        {
            int c = 0;
            for(char ch : str.toCharArray())
            {
                if(allowed.contains(ch + ""))
                    c++;
            }
            if(c == str.length())
                res++;
        }
        return res;
    }
}

File: Coding in Python/13th March 2026/Count the Number of Consistent Strings.py
#Count the Number of Consistent Strings
class Solution:
    def countConsistentStrings(self, allowed: str, words: List[str]) -> int:
        result = 0
        for s in words:
            c = 0
            for ch in s:
                if ch in allowed:
                    c += 1
            if c == len(s):
                result += 1
        return result
